interfaces/__template__/v0/interface_tests/README.md

```diff
@@ -10,6 +10,7 @@ Copy this test to `test_provider.py` and fill in the blanks however appropriate
 
 ```python
 from scenario import Relation, State
+from scenario.context import CharmEvents
```

Comment:
I guess we need https://github.com/canonical/ops-scenario/pull/200 to be released before we simplify this to `from scenario import CharmEvents`. Wonder if we should also rename to `Events` while we're doing that? If people do use this, `CharmEvents` is a bit of a mouthful (and what other kind of events would they be)?

Reply:
> I guess we need [canonical/ops-scenario#200](https://github.com/canonical/ops-scenario/pull/200) to be released before we simplify this to `from scenario import CharmEvents`.

Yes. There's a lot of moving pieces at the moment in terms of releases that interconnect with Scenario, so I was trying to keep things simple by using what's already released. For one thing, pretty soon this should probably be `from ops import testing` with `testing.CharmEvents`.

> Wonder if we should also rename to `Events` while we're doing that? If people do use this, `CharmEvents` is a bit of a mouthful

I think pytest-interface-tests and the related bits in charm-relation-interfaces need to have a deeper look to see how they should work after the Scenario 7 changes (and the exposure in ops.testing). It could have been done as part of the Scenario 7 work, but I only caught on to the need late in the process and didn't want to delay the main work.

I'm not convinced that importing `CharmEvents` (with whatever name, from whatever namespace) is actually the right way to do this (but nor am I convinced it should be strings).

Renaming `scenario.CharmEvents` to `scenario.Events` would be a backwards incompatible change, so we'd have to leave an alias. Renaming `ops.testing.CharmEvents` to `ops.testing.Events` would be ok, but then strangely different from ops-scenario when everything else is the same.

`Events` looks a lot like `_Event`, which I don't like.

> (and what other kind of events would they be)?

One could also ask this about `ops.CharmEvents` :wink:. More seriously, they *aren't* custom events (yet?), and they aren't all types of "ops lifecycle" event (the two collect-status ones are there, a bit inconsistently).

My preference would be to leave this alone for now, and reconsider renaming once we figure out what the API should look like (I feel like we've missed the ideal window for changing the name, so a bit more time is fine).

Reply:
> My preference would be to leave this alone for now

Yep, your rationale makes sense, thanks.